tests/workspaces.rs

```diff
@@ -0,0 +1,684 @@
+extern crate cargotest;
+extern crate hamcrest;
+
+use cargotest::support::{project, execs};
+use cargotest::support::registry::Package;
+use hamcrest::{assert_that, existing_file, is_not};
+
+#[test]
+fn simple_explicit() {
+    let p = project("foo")
+        .file("Cargo.toml", r#"
+            [project]
+            name = "foo"
+            version = "0.1.0"
+            authors = []
+
+            [workspace]
+            members = ["bar"]
+        "#)
+        .file("src/main.rs", "fn main() {}")
+        .file("bar/Cargo.toml", r#"
+            [project]
+            name = "bar"
+            version = "0.1.0"
+            authors = []
+            workspace = ".."
```

Comment:
This 'workspace' key is under `project`, but the RFC [says it goes under 'package'](https://github.com/rust-lang/rfcs/blob/master/text/1525-cargo-workspace.md#new-manifest-keys). Why the discrepancy?